strr-web/package.json

```diff
@@ -44,7 +45,7 @@
     "sass-loader": "^13.3.3",
     "typescript": "5.3.3",
     "vite": "5.4.6",
-    "vitest": "2.1.1",
+    "vitest": "1.3.1",
```

Comment:
That is a significant drop in **vitest** version, why?

Reply:
I reverted back to what it was prior my recent PR.